### PR TITLE
ini parser

### LINKED ISSUE #11: ini parser
- [ ] Write a ini format parser https://en.wikipedia.org/wiki/INI_file
- [ ] Test it to make sure it works correctly.

### PR DESCRIPTION
**Pull request by dgarroDC**
closes #11

### CODE REVIEW

**Comment by araml** [comment]
formatealo

File: src/ini/ini.cpp [hunk 1]
```diff
@@ -0,0 +1,51 @@
+#include <error.h>
+#include <ini.h>
+
+#include <string>
+#include <fstream>
+
+ini::ini(const std::string &path) {
+    std::ifstream ini_file(path);
```
Comment (Saeuissimus) [inline src/ini/ini.cpp]: I would add a TODO here for future reference about unsetting the [sync_with_stdio](https://en.cppreference.com/w/cpp/io/ios_base/sync_with_stdio) flag for the ini stream.

Comment (dgarroDC) [reply src/ini/ini.cpp]: Done.

**Comment by dgarroDC** [comment]
> formatealo

Done.

**Comment by araml** [comment]
Merged.

**Pull request merged**